README.md

```diff
@@ -1,43 +1,34 @@
-# Zowe documentation
+# Zowe™ documentation <!-- omit in toc -->
 
-Welcome to the repo for the Zowe documentation! This repo is the source for [https://docs.zowe.org](https://docs.zowe.org). 
+Welcome to the Zowe documentation repository! This repo is the source for [https://docs.zowe.org](https://docs.zowe.org), also known as "Zowe Docs"!
 
-Zowe documentation is completely open source and we appreciate contributions from the community.
-
-## Table of contents
+Zowe documentation is completely open-source and we appreciate contributions from the community.
 
+- [Table of contents](#table-of-contents)
```

Comment:
You may want to omit the ToC in the Toc 🤣

Reply:
Thanks haha, I just removed the heading because I think it is obvious that it's a ToC :P